src/main/java/seedu/address/logic/commands/iteration/AddIterationCommand.java

```diff
@@ -63,9 +66,19 @@ public CommandResult execute(Model model) throws CommandException {
                     activeCommission.getTitle().toString()));
         }
 
-        activeCommission.addIteration(toAdd);
+        String src = toAdd.getImagePath().path;
+        String dst = storage[0].saveIterationImage(src);
+
+        Iteration toAdd2 = new Iteration(
```

Comment:
Should we not use numbers to distinguish variables? ([Code Quality: Use Name to explain](https://nus-cs2103-ay2223s1.github.io/website/se-book-adapted/chapters/codeQuality.html#use-name-to-explain))